services/reference/linea/json-rpc-methods/eth_call.mdx

```diff
@@ -1,40 +1,11 @@
 ---
 title: "eth_call"
 ---
+import ParserOpenRPC from "@site/src/components/ParserOpenRPC"
+import { NETWORK_NAMES } from "@site/src/plugins/plugin-json-rpc"
 
-import Tabs from '@theme/Tabs';
-import TabItem from '@theme/TabItem';
+<ParserOpenRPC
+  network={NETWORK_NAMES.linea}
+  method="eth_call"
+/>
```

Comment:
This is required 
<img width="555" alt="image" src="https://github.com/user-attachments/assets/adef52d7-91de-41c0-a805-f7728a5d472e">